template.go

```diff
@@ -201,23 +205,50 @@ func (t *Template) Reset(template, startTag, endTag string) error {
 	}
 
 	for {
-		n := bytes.Index(s, a)
+		n := bytes.Index(templateBytes, startTagBytes)
```

Comment:
what are you trying to achive in this  loop
i guess add some guide comment in this `for {}` better

Reply:
ok

Reply:
updated